vercel.json

```diff
@@ -5,6 +5,16 @@
       "destination": "https://graphql-conf-attendee-nextjs.vercel.app/:path*",
       "statusCode": 200
     },
+    {
+      "source": "/conf/gallery",
+      "destination": "https://www.flickr.com/photos/linuxfoundation/albums/72177720320386221/",
+      "permanent": true
+    },
+    {
+      "source": "/conf/gallery/",
+      "destination": "https://www.flickr.com/photos/linuxfoundation/albums/72177720320386221/",
+      "permanent": true
+    },
```

Comment:
Should these be `/conf/2024/gallery` instead?

Reply:
we have 2 galleries at the moment, one for 2023, and another for 2024, I think it makes sense to redirect to the new one yeah.

Reply:
/conf/gallery is 404'ing in the deploy preview, will the redirects work after deploy?

Reply:
> the default should always redirect to the latest conf, in that case 2024, so I think we should leave it as is /conf/gallery.

Makes sense; but we should have a permalink of `/conf/2024/gallery` too IMO; at the moment we have:

https://graphql-github-io-git-fork-yassi-fb0c5c-the-graph-ql-foundation.vercel.app/conf/2024/gallery/ - 404

https://graphql-github-io-git-fork-yassi-fb0c5c-the-graph-ql-foundation.vercel.app/conf/gallery/ - redirects to Flickr 2024

Reply:
> /conf/gallery is 404'ing in the deploy preview, will the redirects work after deploy?

oh I'll fix it, it does seem to be broken indeed

Reply:
> Makes sense; but we should have a permalink of /conf/2024/gallery too IMO

yeah, agree! I'll add it